mkosi/__main__.py

```diff
@@ -9,8 +9,8 @@
 
 from mkosi import load_args, run_verb
 from mkosi.config import MkosiConfigParser
-from mkosi.log import ARG_DEBUG, MkosiPrinter, die
-from mkosi.run import excepthook
+from mkosi.log import ARG_DEBUG, MkosiError, MkosiPrinter, die
+from mkosi.run import excepthook, RemoteException
```

Comment:
```suggestion
from mkosi.run import RemoteException, excepthook
```
This is what isort is complaining about in CI.